static/templates/settings/admin_linkifier_list.hbs

```diff
@@ -9,7 +9,7 @@
     {{#if ../can_modify}}
     <td class="no-select actions">
         <button class="button small delete btn-danger" data-linkifier-id="{{id}}">
-            <i class="fa fa-trash-o" aria-hidden="true"></i>
+            <i class="fa fa-trash-o" data-tippy-content="{{t "Delete" }}" aria-label="{{t "Delete" }}" title="{{t "Delete" }}"></i>
```

Comment:
You don't need both `title` and a tooltip; additionally, when I hover over this button, nothing happens.

It's really important to test even tiny changes like this, because even though it's very simple, a rule of software is that essentially nothing works that you haven't tested :)

Reply:
From https://github.com/zulip/zulip/pull/17045#discussion_r616275251, I assumed that we need both `title` and `data-tippy-content` attribute.
And sorry for the result, actually, I cross-checked the changes and it was working fine for me; the reason that why it was not showing to you was that: since I added the `title` attribute to the `fa fa` icon and not to the `button` therefore it needed hovering exactly over the icon (hovering over the button and not over the icon doesn't give the result).